packages/ui/src/utils/ThemeUtil.ts

```diff
@@ -750,6 +833,14 @@ export const colorStyles = css`
     color: var(--wui-color-fg-300);
   }
 
+  .wui-color-fg-325 {
+    color: var(--wui-color-fg-325);
```

Comment:
This `325` sounds something new. It's from the new design system or old one? If new one, is it okay to have it together with old values?

Reply:
Okay saw your previous comment

Reply:
Yeah i wanted to have it with old values, but wasn't sure if it was going to impact the whole modal UI so kept it simple.